ethabi/src/encoder.rs

```diff
@@ -296,8 +332,8 @@ mod tests {
 		let expected = hex!("
 			0000000000000000000000000000000000000000000000000000000000000020
 			0000000000000000000000000000000000000000000000000000000000000002
-			0000000000000000000000000000000000000000000000000000000000000080
-			00000000000000000000000000000000000000000000000000000000000000e0
+			0000000000000000000000000000000000000000000000000000000000000040
```

Comment:
Why has the encoding of dynamic arrays changed? Was there a bug in the original code? If not, this shouldn't actually be affected by the tuple work.

Reply:
Ooops I should have left note here.  There was indeed a bug in the original code, tests were failing and with the cli one couldn't successfully go back and forth between encoded and decoded data.
 I just confirmed that the same updates have been applied to paritytech/master and the tests pass in both places.

Reply:
Ah, thanks! That's good to know. We should rebase our `graph-patches` on top of their master soon to make sure we pull in updates from them as well. But not this week. 😄